packages/@lwc/engine-core/src/framework/base-lightning-element.ts

```diff
@@ -531,16 +551,16 @@ const queryMethods = [
     'querySelector',
     'querySelectorAll',
 ] as const;
-function getQueryMethod(methodName: typeof queryMethods[number]) {
+function getQueryMethod(methodName: typeof queryMethods[number], renderer: RendererAPI) {
```

Comment:
this is the counter part of the previous note @nolanlawson, also can be removed.